plugin-flex-ts-template-v2/package-lock.json

```diff
@@ -10332,16 +10332,18 @@
       "integrity": "sha512-jPewueSBmM1mffhlFcu3lOgJKINaJNVpeMCzZ31sYOLnp/Ghi9GUwYyRktoEblAetJii+ydSPQTIdiPHe9JDXQ=="
     },
     "node_modules/@twilio/flex-sdk": {
-      "version": "2.2.7",
-      "resolved": "https://registry.npmjs.org/@twilio/flex-sdk/-/flex-sdk-2.2.7.tgz",
-      "integrity": "sha512-Vdw17Sn5HB7CxMcyVslkf1M4QVwLVhLoTOu2vKn8A+KLyba8NCdJPDLUkFfB16btQI3U8JdfGVzXzBk5uWA3gw==",
+      "version": "0.99.3-dev",
```

Comment:
This got my attention for a moment, but I believe this version was set automatically?
I checked this version on npm and it seems to be recent at least, just concerned about it being dev/experimental